data-processing-lib/python/src/data_processing/utils/transform_configurator.py

```diff
@@ -0,0 +1,117 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+from enum import Enum
+from data_processing.utils import get_logger
+
+
+logger = get_logger(__name__)
+
+
+# Supported runtimes
+class TransformRuntime(Enum):
+    """
+    Supported runtimes and extensions
+    """
+    PYTHON = "python"
+    RAY = "ray"
+    SPARK = "spark"
+
+
+class TransformsConfiguration:
+    """
+    Configurations for existing transforms
+    """
+    def __init__(self):
+        """
+        Initialization - building transforms dictionary
+        """
+        self.transforms = {
+            "code2parquet": ("transforms/code/code2parquet/",       # Subdirectory in the project
```

Comment:
The approach here is very delicate and only supports transforms that are part of the repo.  What if we restructure things to move them out of the repo in the future, then this becomes non-functional.  I suppose we can point to other repos instead of subdirs, but having an inventory like this does not seem correct.  Maybe an alternate way of configuring/populating this registry would be from a json file which is then more adaptable.

Reply:
Sure, easy fix

Reply:
done